observation_portal/requestgroups/serializers.py

```diff
@@ -469,6 +469,34 @@ def validate(self, data):
                     'You may only specify a repeat_duration for REPEAT_* type configurations.'
                 ))
 
+        # Validate dither pattern
+
+        # First check that any dither pattern that is set is valid
+        dither_pattern_is_set = False
+        if 'extra_params' in data and 'dither_pattern' in data['extra_params']:
```

Comment:
Should we also make sure that they don't set `dither_pattern` if the request doesn't have any offset values within it?

Reply:
Good idea, I'll add that